frontend/public/locales/en/renew-ita.json

```diff
@@ -199,5 +199,52 @@
     "error-message": {
       "dental-insurance-required": "Select whether you have access to dental insurance"
     }
+  },
+  "review-information": {
+    "read-carefully": "Review this information carefully. The information provided will be used to update your Canadian Dental Care Plan member account.",
+    "page-title": "Review your information",
+    "page-sub-title": "Member information",
+    "dob-title": "Date of birth",
+    "dob-change": "Change date of birth",
+    "client-number-title": "Client number",
+    "client-number-change": "Change client number",
```

Comment:
Since the user authenticates with their client number, they shouldn't be given the ability to change their client number.

Same for their name and date of birth.

Reply:
`dob-change` is still needed for the partner-information section, otherwise the rest has been removed.